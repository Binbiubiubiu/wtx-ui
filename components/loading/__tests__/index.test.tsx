import React from 'react';
import { render } from 'enzyme';
import Loading from '..';
import toJson from 'enzyme-to-json';

describe('Loading', () => {
  it('should render Loading components', () => {
    const wrapper = render(
      <>
        <Loading icon="balls" />
        <Loading icon="bars" />
      </>,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
