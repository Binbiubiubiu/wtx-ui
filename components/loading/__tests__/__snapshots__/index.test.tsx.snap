// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render Loading components 1`] = `
Array [
  <div />,
  <div />,
]
`;
